apps/io-profile/host.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "2.0",
+  "logging": {
+    "logLevel": {
+      "default": "Information",
+      "DurableTask.AzureStorage": "Warning",
+      "DurableTask.Core": "Warning",
+      "Function.GetMessage": "Warning",
+      "Function.GetMessages": "Warning",
+      "Function.GetProfile": "Warning",
+      "Function.GetService": "Warning",
+      "Function.GetUserDataProcessing": "Warning"
+    },
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "maxTelemetryItemsPerSecond": 5,
+        "includedTypes": "PageView;Trace;Dependency;Request",
+        "excludedTypes": "Exception;Event;CustomEvent"
+      }
+    }
+  },
+  "extensions": {
+    "durableTask": {
+      "hubName": "%SLOT_TASK_HUBNAME%",
+      "localRpcEndpointEnabled": false,
+      "storageProvider": {
+        "connectionStringName": "DURABLE_FUNCTION_STORAGE_CONNECTION_STRING"
```

Comment:
It seems the module added it natively, I think we can remove it. See [here](https://github.com/pagopa/dx/blob/daf0a5a0ebd7e8490473625b1d9d7e5907aa465e/infra/modules/azure_function_app/function_app.tf#L60C96-L60C119)
cc @arcogabbo